x/twap/logic.go

```diff
@@ -53,7 +56,7 @@ func (k Keeper) updateRecords(ctx sdk.Context, poolId uint64) error {
 	return nil
 }
 
-// mutates record argument, but not with all the changes.
+// updateRecord mutates record argument, but not with all the changes.
 // Use the return value, and drop usage of the argument.
 func (k Keeper) updateRecord(ctx sdk.Context, record types.TwapRecord) types.TwapRecord {
```

Comment:
I read this a couple times and can't quite figure out what we mean by "but not with all the changes". Can we be more specific?